src/delimiter.js

```diff
@@ -331,74 +373,44 @@ const makeStackedDelim = function(
     // Calculate the depth
     const depth = realHeightTotal / 2 - axisHeight;
 
-    // This function differs from the TeX procedure in one way.
-    // We shift each repeat element downwards by 0.005em, to prevent a gap
-    // due to browser floating point rounding error.
-    // Then, at the last element-to element joint, we add one extra repeat
-    // element to cover the gap created by the shifts.
-    // Find the shift needed to align the upper end of the extra element at a point
-    // 0.005em above the lower end of the top element.
-    const shiftOfExtraElement = (repeatCount + 1) * 0.005 - repeatHeightTotal;
 
     // Now, we start building the pieces that will go into the vlist
-
-    // Keep a list of the inner pieces
-    const inners = [];
+    // Keep a list of the pieces of the stacked delimiter
+    const stack = [];
 
     // Add the bottom symbol
-    inners.push(makeInner(bottom, font, mode));
+    stack.push(makeCorner(bottom, font, mode));
+    stack.push(lap); // overlap
 
     if (middle === null) {
-        // Add that many symbols
-        for (let i = 0; i < repeatCount; i++) {
-            inners.push(lap); // overlap
-            inners.push(makeInner(repeat, font, mode));
-        }
+        // The middle section will be an SVG. Make it an extra 0.016em tall.
+        // We'll overlap by 0.008em at top and bottom.
+        const innerHeight = realHeightTotal - topHeightTotal - bottomHeightTotal
+            + 0.016;
+        stack.push(makeInner(repeat, innerHeight, options));
     } else {
         // When there is a middle bit, we need the middle part and two repeated
         // sections
-        for (let i = 0; i < repeatCount; i++) {
-            inners.push(lap);
-            inners.push(makeInner(repeat, font, mode));
-        }
-        // Insert one extra repeat element.
-        inners.push({type: "kern", size: shiftOfExtraElement});
-        inners.push(makeInner(repeat, font, mode));
-        inners.push(lap);
+        const innerHeight = (realHeightTotal - topHeightTotal - bottomHeightTotal -
+            middleHeightTotal) / 2 + 0.016;
+        stack.push(makeInner(repeat, innerHeight, options));
```

Comment:
It would be good to have a constant for the overlap to avoid having `const lap = {type: "kern", size: -0.008};` up above and the `+ 0.016` here get out of sync.

Reply:
I'll tie them together.